### ISSUE TITLE
Docs: suggest creating an empty key

### ISSUE THREAD

**Issue by stchris** [alexwennerberg/gourami#15]
While setting up locally (with the `sample_env` you provide) it isn't possible to create an account if the `SIGNATURE_PUBKEY_PEM` file isn't there:

```
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/ap.rs:45:21
```

Since I don't care about federation I side-stepped this with `mkdir local && touch local/public.pem`. You might want to mention this in the docs (or handle the error differently).

**Comment by alexwennerberg** [comment]
Going to fail gracefully if this key is absent with `unwrap_or("".to_owned())`

**Comment by alexwennerberg** [comment]
Resolved in b3db9dc73f26630aa33239132fee206da804dd0e
